apps/voting/app/src/components/VoteDetail/DisputableActionStatus.js

```diff
@@ -0,0 +1,120 @@
+import React from 'react'
+import {
+  Box,
+  Button,
+  GU,
+  IconLock,
+  Info,
+  Link,
+  textStyle,
+  Timer,
+  useTheme,
+} from '@aragon/ui'
+import { format } from 'date-fns'
+import { getAgreement } from '../../agreementsMockData'
+import { addressesEqual } from '../../web3-utils'
+
+const formatDate = date => `${format(date, 'yyyy-MM-dd, HH:mm')}`
+
+function DisputableActionStatus({ vote, connectedAccount }) {
+  //TODO: get agreement and vote real data
+  const theme = useTheme()
+  const agreement = getAgreement()
+  return (
+    <Box heading="Disputable Action Status">
+      <Item>
+        <Label>Status</Label>
+        <div>DisputableStatusTag</div>
+      </Item>
+      <Item>
+        <Label>Action collateral locked</Label>
+        <div
+          css={`
+            display: flex;
+            align-items: center;
+          `}
+        >
+          {vote.disputable.action.collateral.challengeAmount}{' '}
+          {vote.disputable.action.collateral.collateralToken}
+          <span
+            css={`
+              display: inline-flex;
+              padding-left: ${1 * GU}px;
+            `}
+          >
+            <IconLock size="small" />
+          </span>
+        </div>
+      </Item>
+      <Item>
+        <Label>Challenge period</Label>
```

Comment:
Just checking, in the designs this can be either "Settlement period" or "Challenge period" depending on the status, do we need to worry about that now?

Reply:
Yes, I am planning on adding this validation on a later PR :)